slides/snippets/lib/file_snippets.rb

```diff
@@ -86,43 +86,44 @@ def create_textbox_with_text(presentation_id, page_id)
     element_id = 'MyTextBox_01'
     pt350 = {
       magnitude: '350',
-      unit: 'PT'
+      unit:      'PT'
     }
     requests = [{
       create_shape: {
-        object_id_prop: element_id,
-        shape_type: 'TEXT_BOX',
+        object_id_prop:     element_id,
+        shape_type:         'TEXT_BOX',
         element_properties: {
           page_object_id: page_id,
-          size: {
+          size:           {
             height: pt350,
-            width: pt350
+            width:  pt350
           },
-          transform: {
-            scale_x: '1',
-            scale_y: '1',
+          transform:      {
+            scale_x:     '1',
+            scale_y:     '1',
             translate_x: '350',
             translate_y: '100',
-            unit: 'PT'
+            unit:        'PT'
           }
         }
       }
     },
 
-    # Insert text into the box, using the supplied element ID.
-    {
-      insert_text: {
-        object_id_prop: element_id,
-        insertion_index: 0,
-        text: 'New Box Text Inserted!'
-      }
-    }]
+                # Insert text into the box, using the supplied element ID.
```

Comment:
Indenting seems way off here.

Reply:
This is because the opening `{` is on column 17.
If we'd like to change this, we need to change the column of the open curly brace, perhaps moving it to he next line.

Reply:
yeah, see if you can get it to line up better. The two requests in the array should be aligned.